### PR TITLE
fix build of STAR 2.7.11a + 2.7.11b on non-x86 architectures by avoiding use of `-maxv2` + add missing `xxd` build dependency

### PR DESCRIPTION
**Pull request by jpecar**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb [hunk 1]
```diff
@@ -24,6 +24,9 @@ checksums = [
     {'STAR-2.7.11a_use-external-htslib.patch': '2fdc3ed9372d983f77d861d6f16a60a553598358dce9ff8216f96eb20e63ce4e'},
 ]
 
+#by default this is set to -mavx2 which makes it fail on non x86 systems
+prebuildopts = 'CXXFLAGS_SIMD= '
```
Comment (boegel) [inline easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb:28]: @jpecar This should probably be `buildopts` (so we get "`make CXXFLAGS_SIMD=`")

**Review by boegel — CHANGES_REQUESTED** [review]

**Review by boegel — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb (hunk 1 above)
Comment (jpecar) [reply easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb:28]: Actually no. I checked build logs and prebuildopts results in the right thing (no -mavx2) while buildopts does not.

File: easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb [hunk 2]
```diff
@@ -24,6 +24,9 @@ checksums = [
     {'STAR-2.7.11a_use-external-htslib.patch': '2fdc3ed9372d983f77d861d6f16a60a553598358dce9ff8216f96eb20e63ce4e'},
 ]
 
+#by default this is set to -mavx2 which makes it fail on non x86 systems
```
[the remark this answers is not in the document]
Comment (jpecar) [reply easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb]: at least that's easy to fix ;)

File: easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/s/STAR/STAR-2.7.11b-GCC-13.2.0.eb:28]: Verified, works as intended indeed via `prebuildopts`

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
aarch64-neoverse-n1-node1.int.aws-rocky88-202310.eessi.io - Linux Rocky Linux 8.8, AArch64, ARM UNKNOWN (neoverse_n1), Python 3.6.8
See https://gist.github.com/boegel/fbe24ce7c2bcaa8df6d4ee9b50290c03 for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen3

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
node3116.skitty.os - Linux RHEL 8.8, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/boegel/b12477c01b00231d1ee85419c5a14fa0 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @jpecar!

**Pull request merged**